pkg/operation/botanist/dns.go

```diff
@@ -72,8 +67,7 @@ func (b *Botanist) DeployDNSRecord(terraformerPurpose, name, target string, purp
 		}
 		if cname == fmt.Sprintf("%s.", target) {
 			b.Logger.Infof("Skipping DNS record registration because '%s' already points to '%s'", name, target)
-			// Clean up possible existing Terraform job/pod artifacts from previous runs
-			return tf.EnsureCleanedUp()
```

Comment:
Why has this been deleted?

Reply:
No terraformer is being created anymore and every terraformer does it's cleanup anyways before and after every run.